docs/syntax-attributes-functions-parameters.md

```diff
@@ -6,48 +6,56 @@ This RFC proposes a syntax for function attribute parameters. This is a follow u
 
 ## Motivation
 
-The [Attributes](./syntax-attributes-functions.md) RFC provides a syntax for parameterless function attributes. This suffices for some potential use cases, such as `@native`, and `@inline`. However, other potential use cases such as `@deprecated` will benefit from an expressive attribute syntax that allows parameters to be supplied. We can use the parameters to provide the name of the function to be used in place of the deprecated function, and the reason for deprecation. This information can be used to generate informative deprecation warning messages. Another potential use case could be an `@unroll` attribute for loop unrolling which could use a numeric parameter for specifying the number of iterations to be unrolled. This would also require supporting attributes on loops, which would be a topic of discussion for a separate RFC. It might be desirable to allow these attributes to assume a default value for parameters not provided explicitly, but that's a discussion reserved for attribute-specific RFCs.
+The [Attributes](./syntax-attributes-functions.md) RFC provides a syntax for parameterless function attributes. This suffices for some potential use cases, such as `@native`, and `@inline`. However, other potential cases such as `@deprecated` will benefit from an expressive attribute syntax that allows parameters to be supplied. We can use the parameters to provide the name of the function to be used in place of the deprecated function, and the reason for deprecation. This information can be used to generate informative deprecation warning messages. Another potential use case could be an `@unroll` attribute for loop unrolling which could use a numeric parameter for specifying the number of iterations to be unrolled. This would also require supporting attributes on loops, which would be a topic of discussion for a separate RFC. It might be desirable to allow these attributes to assume a default value for parameters not provided explicitly.
 
 ## Design
 
-The following syntax is proposed for attributes with parameters:
+The following syntax is proposed for attributes:
 
 ```ebnf
-attribute = '@' NAME 
-          | '@' NAME '(' ')'
-          | '@' NAME '(' ATOMIC-LITERAL (',' ATOMIC-LITERAL)* ')'
-```
+parameter-table = '{' '}'
+          | '{' parameter (sep parameter)* '}'
```

Comment:
Usual zero-to-many repetition syntax in EBNF is `{}`
```suggestion
| '{' parameter {sep parameter} '}'
```

Reply:
There's also an existing `fieldsep` grammar in Luau for table field separator.

Reply:
The `ebnf` syntax has been reworked entirely and this issue is fixed. I adapted the table grammar from Lua's documentation.